eng/Version.Details.xml

```diff
@@ -57,11 +57,11 @@
       <Uri>https://github.com/aspnet/Extensions</Uri>
       <Sha>54d51a340698b6883dd3e47be372c07e0acf75bc</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Sdk" Version="3.0.100-preview5.19303.1">
+    <Dependency Name="Microsoft.NET.Sdk" Version="3.0.100-preview6.19303.1" CoherentParentDependency="Microsoft.Dotnet.Toolset.Internal">
```

Comment:
Why the manual change from preview5 to preview6 here?

Reply:
`preview5` was an error from manual updates (nothing in source-build looks at these version numbers yet).  The 19303.1 build was always a `preview6` tag.